### PR TITLE
Drop coding lines from all python files.

### PR DESCRIPTION
**Pull request by alunduil**

### CODE REVIEW

**Comment by alunduil** [comment]
The switch to dataclasses is dropping Python 3.6 sooner than I was hoping but will be fine.

File: zfs_test/replicate_test/task_test/report_test.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""zfs.replicate.task.report tests."""
```
Comment (Flameeyes) [inline zfs_test/replicate_test/task_test/report_test.py:1]: Unrelated file change?

**Review by Flameeyes — APPROVED** [review]

File: zfs_test/replicate_test/task_test/report_test.py (hunk 1 above)
Comment (alunduil) [reply zfs_test/replicate_test/task_test/report_test.py:1]: Added to keep test coverage happy. Apparently the coding lines provided noticeable covered lines. :p
